### PR TITLE
Removed Goerli from the autogenerated README.md

### PR DESCRIPTION
**Pull request by qbzzt**
1. Closing: ENG-1382 (remove Goerli from autogenerated README.md)
2. Fixed formatting
3. Renamed `OVM_L1ETHGateway` to `PROXY__OVM_L1ETHGateway`

### CODE REVIEW

File: packages/contracts/scripts/generate-markdown.js [hunk 1]
```diff
@@ -74,7 +82,7 @@ const networks = {
       if (contracts[i] === 'OVM_L1CrossDomainMessenger') {
         proxiedContracts.push(contracts.splice(i, 1)[0])
       }
-      if (contracts[i] === 'OVM_L1ETHGateway') {
+      if (contracts[i] === 'PROXY__OVM_L1ETHGateway') {
```
Comment (smartcontracts) [inline packages/contracts/scripts/generate-markdown.js:85]: What is this change for?

**Review by smartcontracts — COMMENTED** [review]
Please lint this file and run the script to regenerate the markdown file and commit the result

File: packages/contracts/scripts/generate-markdown.js (hunk 1 above)
Comment (qbzzt) [reply packages/contracts/scripts/generate-markdown.js:85]: @elenadimitrova  said she renamed the contract.

Comment (smartcontracts) [reply packages/contracts/scripts/generate-markdown.js:85]: Oh I think the correct diff here is `OVM_L1ETHGateway` => `OVM_L1StandardBridge`

**Comment by tynes** [comment]
Small nit - prefix the commit messages with the package being modified like `contracts: removed goerli...`

**Comment by smartcontracts** [comment]
@qbzzt can you rebase this on top of latest experimental? We had a mishap yesterday so all open PRs against experimental need to be rebased.

**Comment by qbzzt** [comment]
Replaced by #1454

**Comment by qbzzt** [comment]
> @qbzzt can you rebase this on top of latest experimental? We had a mishap yesterday so all open PRs against experimental need to be rebased.

I closed the PR and created a clean one: #1454

**Pull request closed without merging**